main.go

```diff
@@ -37,6 +37,7 @@ var (
 	timeGranularity = flag.String("granularity", "day", "how often we expect to append new data")
 	streamStart     = flag.String("streamStart", "", "The start of the streamed data. Only used if granularity='stream'. Used to ensure idempotency")
 	streamEnd       = flag.String("streamEnd", "", "The end of the streamed data. Only used if granularity='stream'. Used to ensure idempotency")
+	isTargetPST     = flag.Bool("targetPST", false, "Whether or not the target data time column is in PST.")
```

Comment:
PST and PT are not quite the same: https://en.wikipedia.org/wiki/Pacific_Time_Zone
I'd use PT in every instance here, for clarity.